src/go/rpk/pkg/cli/cmd/plugin/plugin.go

```diff
@@ -294,3 +304,35 @@ func determineBinDir() (string, error) {
 	}
 	return paths[0], nil
 }
+
+func tryDirectDownload(name string) ([]byte, error) {
+	u := fmt.Sprintf("https://packages.vectorized.io/cloudv2/raw/names/rpk-plugin-%s-%s-%s/versions/0.0.3/rpk-plugin-%[1]s-%[2]s-%[3]s", name, runtime.GOOS, runtime.GOARCH)
```

Comment:
why is 0.0.3 hardcoded in the URL?

Reply:
I think until we get a manifest for cloudsmith, we should just ovewrite the binary in place (and be versionless). cc @ivotron about removing the version from the path.

Reply:
that's done automatically done by cloudsmith

Reply:
Ok, we don't have versions without a manifest. We can add a hidden --version flag for now and use it in the path. cc @r-vasquez and @ivotron what do you think?

Reply:
I like it as a temporal solution until we get the manifest from Cloudsmith but... what would be the default version?

Reply:
default version "", if no version specified, don't try cloudsmith